2_deploy_ses_aio/play.yml

```diff
@@ -0,0 +1,74 @@
+#
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+- name: Install SES 5
+  hosts: ses_nodes:ses
+  remote_user: root
+  gather_facts: true
+  roles:
+    - ses
+  vars:
+    ses_openstack_config: True
+
+- hosts: ses_nodes[0]
+  vars:
+    ses_config_file_path: "~/suse-osh-deploy/ses_config.yml"
+  tasks:
+    - name: Ensure the OSH config folder exists
+      file:
+        state: directory
+        name: "{{ ses_config_file_path | dirname }}"
+      delegate_to: localhost
+
+    # The AIO ses creates /tmp/ses_config/ on localhost,
```

Comment:
Should we actually follow this? As right now (with https://github.com/SUSE-Cloud/socok8s/pull/67) we're generating SES config ourselves, we could drop it directly under  `ses_config_file_path`, right?

Reply:
I believe this could be considered as something to quickly merge.

We can iterate with the results of #67 when we are ready.

I agree with you, this needs cleanup too, though.

Reply:
that makes sense if that is changed and merged, will happily rebase once that gets in.

Reply:
We may even integrate that PR with the play.yml file completely as the setup_ses_config doesnt seem like a proper role to me, or move everything into a ses-osh role and include it from the play.yml file